docs/notebooks/neural_dual.ipynb

```diff
@@ -12,7 +12,7 @@
    "metadata": {},
```

Comment:
<p>Maybe it would be worthwhile to also keep the previous approach where <code>neural_g</code> is an <code>ICNN</code> , e.g., as:</p><p><code>num_train_iters = 10_000</code></p><p><code>g_as_icnn = False</code></p><p> </p><p><code>neural_f = icnn.ICNN(...)</code></p><p><code>if g_as_icnn:</code></p><p><code>    neural_g = icnn.ICNN(...)</code></p><p><code>else:</code></p><p><code>    neural_g = mlp.PotentialMLP(...)</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/219/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>I think switching back to an ICNN for <code>g</code> doesn't work very well right now. I can debug a bit when trying to fix the inverse map and see if I can get it doing something reasonable </p>